docs/windows.md

```diff
@@ -218,6 +218,10 @@ sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERV
 kubectl apply -f -
 ```
 
```

Comment:
It's better to move this part to line 209, and we can guide the user on how to run Install-OVS.ps1 to only install the unsigned OVS driver.

Reply:
I agree that the current version is too confusing. This information should be provided to users _before_ they try to apply the yaml to deploy the DaemonSet.

> The above commands also install the signed OVS driver.

This is also confusing. Our upstream Windows Docker image does not include a signed OVS driver, so it seems that this case would never apply to an OSS user reading this documentation?

Reply:
I have made the required changes in the docs